crates/ark/src/variables/r_variables.rs

```diff
@@ -357,6 +386,47 @@ impl RVariables {
         }
     }
 
+    /// Gets the value of the special variable '.Last.value' (the value of the
+    /// last expression evaluated at the top level), if show_last_value is true.
+    ///
+    /// For backward compatibility, also checks the global option "positron.show_last_value"
+    /// if it exists, but the instance variable takes precedence.
+    ///
+    /// Returns None in all other cases.
+    fn last_value(&self) -> Option<PositronVariable> {
+        // First check our instance variable
+        let show_last_value = if self.show_last_value {
+            true
+        } else {
+            // If we aren't always showing the last value, update from the
+            // global option
+            let use_last_value = get_option("positron.show_last_value");
+            match use_last_value.get_bool(0) {
+                Ok(Some(true)) => true,
+                _ => false,
+            }
+        };
+
+        if show_last_value {
+            match harp::parse_eval_global(".Last.value") {
+                Ok(last_robj) => Some(PositronVariable::from(
+                    String::from(".Last.value"),
+                    String::from(".Last.value"),
+                    last_robj.sexp,
+                )),
+                Err(err) => {
+                    // This isn't a critical error but would also be very
+                    // unexpected.
+                    log::debug!("Environment: Could not evaluate .Last.value ({err:?})");
```

Comment:
```suggestion
                    log::error!("Environment: Could not evaluate .Last.value ({err:?})");
```

We avoid `debug!` out of habit typically